selfapi.js

```diff
@@ -367,6 +372,21 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  toAPIIndex: function (basePath) {
+    var fullPath = normalizePath(this.path, basePath) || '/';
+    return function () {
+      var routes = {};
+      Object.keys(this.children).forEach(function (child) {
+        if (child == "null") {
+          return;
```

Comment:
Nit: I still don't think we should have this check here. Please remove it. If something bad happens when you do, please paste the error log somewhere, e.g. in an issue for this repo, preferably with a stack trace (e.g. with `console.error(new Error().stack)`).